### PR TITLE
Ftr/8931 unlinked award warning message

### PR DESCRIPTION
**Pull request by brianpetway**
**High level description:**

Make a new component for this warning message and make it conditionally appear in two places on the award page.

**Technical details:**

Added logic to set a bool for the condition, then made the warning message component
Changed the padding for the award profile page at mobile size only, to match our new standard and show more content on the screen.
Url for example to test with is in the ticket.

**JIRA Ticket:**
[DEV-8931](https://federal-spending-transparency.atlassian.net/browse/DEV-8931)

**Mockup:**
https://app.zeplin.io/project/61f806d9ad825610f7b1fc0a/screen/62c440d7a2d3e8118b42d3d8

The following are ALL required for the PR to be merged:

Author:
- [x ] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/award/shared/AwardPageWrapper.jsx [hunk 1]
```diff
@@ -32,7 +31,7 @@ const AwardPageWrapper = ({
         if (!areDefCodesLoading) {
             setCovidDefCodes(defCodes.filter((c) => c.disaster === 'covid_19' && allDefCodes.indexOf(c.code) > -1).map((code) => code.code));
         }
-    }, [areDefCodesLoading, allDefCodes]);
+    }, [areDefCodesLoading, allDefCodes, defCodes]);
```
Comment (nick-torres) [inline src/js/components/award/shared/AwardPageWrapper.jsx:34]: was this addition intentional?

Comment (brianpetway) [reply src/js/components/award/shared/AwardPageWrapper.jsx:34]: no, it was auto-added on save but i left it bc i don't think it's causing problems

**Review by nick-torres — APPROVED** [review]
